src/tests/__snapshots__/home.test.js.snap

```diff
@@ -0,0 +1,47 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
- [ ] You only have snapshot tests for the Calculator and Home components, but you are required to add snapshot tests for all your component. Kindly add snapshot tests for the Layout, Navbar, and Quotes components.